crossgen.cmake

```diff
@@ -1,6 +1,7 @@
 # Contains the crossgen build specific definitions. Included by the leaf crossgen cmake files.
 
 add_definitions(
+    -DFEATURE_PREJIT
```

Comment:
I have not bothered with fixing crossgen to compile with `FEATURE_PREJIT` undefined.